manifests/z/Zoom/Zoom/5.10.0/Zoom.Zoom.locale.en-US.yaml

```
@@ -1,8 +1,7 @@
-# Created with YamlCreate.ps1 v2.1.1 $debug=QUSU.7-2-1
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
 
 PackageIdentifier: Zoom.Zoom
-PackageVersion: 5.10.4306
+PackageVersion: 5.10.0
```

```suggestion
PackageVersion: 5.10.4306
```
This must match Control Panel to prevent any sort of upgrade loops when running the `winget upgrade Zoom.Zoom`.